src/components/Setup/Views/Handle.view.tsx

```
@@ -0,0 +1,429 @@
+                                isKernelClientReady && !isSendingUserOp
+                                    ? "bg-green-500 hover:bg-green-700 focus:ring-green-500"
+                                    : "bg-gray-500"
+                            }`}
+                        >
+                            {isSendingUserOp ? <Spinner /> : "Send UserOp"}
+                        </button>
+                        {/* UserOp Status Label */}
+                        {userOpHash && (
+                            <div
+                                className="mt-4"
+                                dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)